inductiva/templating/renderers.py

```diff
@@ -9,43 +8,8 @@
 from inductiva import types
 
 
-class RendererAdapter:
-    """Abstract base class for template renderers."""
-
-    TEMPLATE_EXTENSION = None
-
-    @abstractmethod
-    def set_template_dir(self, template_dir: types.PathOrStr) -> None:
-        """Set the directory where the templates are located."""
-        pass
-
-    @abstractmethod
-    def render(self, template_name: types.PathOrStr,
-               target_file: Union[types.PathOrStr, IO], **render_args):
-        """Render a file."""
-        pass
-
-    @classmethod
-    def is_template(cls, file: types.PathOrStr) -> bool:
-        """Check if the given file is of template type."""
-        if isinstance(file, pathlib.Path):
-            return file.suffix == cls.TEMPLATE_EXTENSION
-        elif isinstance(file, str):
-            return file.endswith(cls.TEMPLATE_EXTENSION)
-        return False
-
-    @classmethod
-    def strip_extension(cls, file: types.PathOrStr) -> str:
-        """Strip the template extension if the given file is a template."""
-        if not cls.is_template(file):
-            return file
-        if isinstance(file, pathlib.Path):
-            return file.with_suffix("")
-        return os.path.splitext(file)[0]
-
-
-class JinjaRendererAdapter(RendererAdapter):
-    """Adapter for the Jinja2 template engine.
+class JinjaRenderer():
```

Comment:
Not really sure why we need this class.
Shouldn't all these be internal methods of the TemplateManager class?

Reply:
Yes, you are right, I think it would be possible to merge the two classes into one.

Reply:
Please consider doing that in a subsequent PR. Giving LGTM now.
Thank you!